flutter-ory-network/android/app/build.gradle

```
@@ -45,10 +45,10 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "com.example.ory_network_flutter"
+        applicationId "com.ory.flutter_ory_network"
```

```suggestion
        applicationId "sh.ory.flutter_ory_network"
```

Probably needs to be replaced in a bunch of other places.